features/user_registration.feature

```diff
@@ -0,0 +1,14 @@
+Feature: As a visitor
+In order to checkout
+I would like to create an account
+
+
+Scenario: I create an account
```

Comment:
Love it, @sigu. Yes! @casperjureskog this should definitely have a sad path. Try making a new scenario when the user doesn't fill in email. You'll be able to use the built-in devise flash messages to do like `I should see "Email is required"`